ci/jenkins/README.md

```diff
@@ -0,0 +1,28 @@
+## Antrea CI: Jenkins
+
+### Reasons for Jenkins
+We have tests as Github Action but Jenkins allows tests running on a cluster of multiple nodes and offers better environment setup options.
+
+### List of Jenkins jobs
+* [e2e](https://github.com/vmware-tanzu/antrea/tree/master/test/e2e): end-to-end tests for Antrea.
+* conformance: community tests using sonobuoy, focusing on "Conformance" and skip "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage", "sig-auth", "sig-api-machinery", "sig-apps", or "sig-node".
+* network policy: community tests using sonobuoy, focusing on "Feature:NetworkPolicy" and skip "allow ingress access from updated pod" or "named port".
```

Comment:
s/skip/skipping
Please add a pointer to the named port Antrea issue (https://github.com/vmware-tanzu/antrea/issues/122) to explain why we are skipping "named port".
Please add a pointer to https://github.com/kubernetes/kubernetes/issues/85908 to explain why we are skipping "allow ingress access from updated pod"

Reply:
changed "skip" to "skipping". Two links added. Thanks.